content/03.heterogeneity.md

```diff
@@ -1,26 +1,26 @@
+Other approaches like reprocessing the data using a single pipeline (when data are obtained from multiple sources), using batch correction methods [@doi:10.1093/biostatistics/kxj037; @doi:10.1093/nar/gku864], and normalizing raw values [@doi:10.1186/gb-2010-11-3-r25] may be necessary to obtain meaningful insights from the data. 
 
-Dimensionality reduction, or more fundamentally representation learning, learns low-dimensional representations or composites of features from the raw data. 
-For example, representation learning through matrix factorization can extract composite features from transcriptomics datasets that are made of combinations of gene expression levels found in the training data[@doi:10.1038/s41467-020-14666-6], and then use them to interpret test input data [@doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3].
-To ensure that the learned representations are generalizable, the features learned by the model can be constrained through methods like regularization [@doi:10.1371/journal.pgen.1004754, @doi:10.1002/sim.6782]. 
-Representation learning generally requires many samples in complex biological systems and therefore may appear to aggravate the curse of dimensionality. 
+Dimensionality reduction, or more fundamentally, representation learning, learns low-dimensional representations (composite features) from the raw data. 
+For example, representation learning through matrix factorization can extract features from transcriptomics datasets that are made of combinations of gene expression values found in the training data [@doi:10.1038/s41467-020-14666-6], and use them to interpret test data [@doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3].
+To ensure that the learned representations are generalizable to other data, the features learned by the model can be constrained through methods like regularization [@doi:10.1371/journal.pgen.1004754, @doi:10.1002/sim.6782]. 
+Representation learning generally requires many samples when applied to complex biological systems and therefore may appear to aggravate the curse of dimensionality. 
 However, it can be a powerful tool to learn low-dimensional patterns from large datasets and then find those patterns in smaller, related datasets. 
-In the later sections of this perspective, we will discuss this method of leveraging large datasets to reduce dimensionality in smaller datasets, also known as feature-representation-transfer. 
+In later sections, we will discuss this method of leveraging large datasets to reduce dimensionality in smaller datasets, also known as feature-representation-transfer learning. 
 
 ![Dimension reduction can help manage the curse of dimensionality in rare disease data](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/content/images/figures/pdfs/dimensionality-reduction.pdf){#fig:1}
```

Comment:
Do you want to expand this figure legend a bit? Is it worth adding lettering to this figure so you can cite it in the text at multiple points?